build-system/build.conf.js

```diff
@@ -35,6 +36,11 @@ module.exports = {
         }],
       ]);
     }
-    return defaultPlugins;
+    if (isCommonJsModule) {
+      pluginsToApply = pluginsToApply.concat([
+        [require.resolve('babel-plugin-transform-commonjs-es2015-modules')],
```

Comment:
Do we have an open issue for removing this transform when possible? We should migrate over all these dependencies to ESM.

Reply:
I have a TODO in the code assigned to @erwinmombay and me. I'm working on a follow up PR to upgrade to the latest closure as we speak, and will file issues for all remaining ESM library related tasks.

Reply:
Makes sense, thanks @rsimha.

Reply:
Sent out #23106 to clean this up.